<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mmind::ProductInfoSettingsPage</class>
 <widget class="QWidget" name="mmind::ProductInfoSettingsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>329</width>
    <height>146</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="verticalSpacing">
    <number>10</number>
   </property>
   <item row="2" column="0">
    <widget class="QPushButton" name="pdfPath">
     <property name="text">
      <string>PDF上传</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>制品</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>车型</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="model"/>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="comboBox">
     <item>
      <property name="text">
       <string>FrSus</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>RrSus</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="pdf">
     <property name="text">
      <string>D:/123.pdf</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="excel">
     <property name="text">
      <string>D:/123.xlsx</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="excelPath">
     <property name="text">
      <string>报表模板</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
